src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs

```diff
@@ -60,15 +61,13 @@ public enum Metrics
         internal const string LoadName = "MultiClassClassifierEvaluator";
 
         private readonly int? _outputTopKAcc;
-        private readonly bool _names;
```

Comment:
So it seems this variable is unused, so you've deleted it. But I'd prefer to keep it, perhaps for historic reasons, in particular because the arguments of the Name Argument bellow which is used to initialize the _names field. This argument might still be reachable through external APIs (like NimbusML), and even if it's not doing anything right now, I'd prefer to keep the `_names` member for historical reasons, if it at anytime becomes relevant when investigating something down the road.

https://github.com/dotnet/machinelearning/blob/a9ab7fc530ba9a2a519aaca70862ee67f42bf712/src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs#L37-L38